class/practice7.1/MaybeTheBestCodeEver.cpp

```diff
@@ -0,0 +1,56 @@
+#include <iostream>
+#include <print>
+#include "Rectangle.hpp"
+
+int main() {
+    double widths[5] = {0.0};
+    double heights[5] = {0.0};
+    int count = 0;
+    double biggestArea = 0;
+    double smallestArea = 1000000;
+
+    for (int i = 0; i < 5; ++i) {
```

Comment:
here and below we are using hardcoded value 5. It is better to use named constant ot size of container instead